recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/Theano/libgpuarray.git
+  git_tag: v0.6.0
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - git
+    - cmake
+    - python
+    - cython >=0.25
+    - numpy
+    - mako
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - mako
+    - six
+    - nose  # This will be fixed in 0.6.1
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgpuarray.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libgpuarray.so  # [linux]
+    - if not exists %PREFIX%\\Library\\lib\\libgpuarray.lib exit 1  # [win]
+  imports:
+    - pygpu
+    - pygpu.gpuarray
+
+about:
+  home: http://github.com/Theano/libgpuarray
+  license: BSD
```

Comment:
Please include the version (e.g. `BSD 3-Clause` for the 3-Clause BSD license).

Note: After reading the license file this looks like it might be ISC. Is that intentional? That wouldn't cause us any issues packaging it. Just trying to see whether this should be updated or not.